### PR TITLE
Twilio message payload

### PR DESCRIPTION
**Pull request by sabyasachi600**
Creation of message payload.

### CODE REVIEW

File: modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package exv.twilio.plugin.messaging
+/**
+ * Created with IntelliJ IDEA.
```
Comment (kupaul) [inline modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs:3]: Delete these commented lines

File: modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * User: 91990
+ * Date: 17/4/20
+ * Time: 1:01 PM
+ * To change this template use File | Settings | File Templates.
+ */
+class TwilioIntegrationPayload {
```
Comment (kupaul) [inline modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs:9]: Add some class level comment. Like what is this class used for

File: modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    function  MessagePayloadCreation() : String {
```
Comment (kupaul) [inline modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs:10]: Write method level comment. Like what this method is doing, input parameter and return type

File: modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs (hunk 1 above)
Comment (sabyasachi600) [reply modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs:3]: okk

File: modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs (hunk 2 above)
Comment (sabyasachi600) [reply modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs:9]: okk

File: modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs (hunk 3 above)
Comment (sabyasachi600) [reply modules/configuration/gsrc/exv/twilio/plugin/messaging/TwilioIntegrationPayload.gs:10]: okk

**Pull request merged**